.github/workflows/ci.yml

```
@@ -64,29 +64,32 @@ jobs:
         with:
           flags: cypress
   s3-deploy:
-    name: S3 Deploy
+    name: Deploy to Amazon S3
     needs:
       - build_test
       - cypress
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '16'
-      - name: Setup Ruby
-        uses: actions/setup-ruby@v1
+      - uses: actions/setup-node@v2
         with:
-          ruby-version: '2.7'
+          node-version: "16"
       - name: Install Dependencies
-        run: |
-          gem install s3_website -v 3.4.0
-          npm ci
-      - name: Build
-        run: npm run build
-      - name: Deploy
-        run: ./s3_deploy.sh
+        run: npm ci
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          AWS_EC2_METADATA_DISABLED: true
+          # Skip installing Cypress since it isn't needed for just building
+          # This decreases the deploy time quite a bit
+          CYPRESS_INSTALL_BINARY: 0
+      - name: Deploy files to Amazon S3
+        uses: concord-consortium/s3-deploy-action@v1
+        with:
+          bucket: models-resources
+          prefix: ${{ github.event.repository.name }} # 'question-interactives'
+          awsAccessKeyId: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          # Add the default branch and the current branch as top branches
+          topBranches: |-
+            [
+              "${{ github.event.repository.default_branch }}",
+              "${{ github.ref_name }}"
+            ]
```

```suggestion
          # Add the default branch as a top branch
          topBranches: |-
            [
              "${{ github.event.repository.default_branch }}"
            ]
```
Before merging the current branch should be removed, so we don't end up with separate `index-[branch]`.html` files for each branch going forward.